CardiorespiratoryFitness/CardiorespiratoryFitness/iOS/CRFCameraSettings.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  CRFCameraSettings.swift
+//  CardiorespiratoryFitness
+//
+//  Copyright © 2017-2018 Sage Bionetworks. All rights reserved.
```

Comment:
just 2018 for a new file? or is this Github getting confused about a file rename?